test/Bugs/inbug.js

```diff
@@ -0,0 +1,41 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+function test0() {
+  var obj0 = {};

```

Comment:
I think it’s better to write simple test instead of Exprgen file.
We can one test for each condition and only print fail if it fail so that there is no need for a baseline file.

Reply:
Thanks, I cleaned it up.